contrib/gocql/gocql/gocql.go

```diff
@@ -86,6 +86,15 @@ func (tq *Query) newChildSpan(ctx context.Context) ddtrace.Span {
 	return span
 }
 
+// FinishChildSpan ends child span
+func (tq *Query) finishChildSpan(span ddtrace.Span, err error) {
+	if tq.params.config.noDebugStack {
```

Comment:
Can we make it a bit less repetitive? (even if potentially uglier)
```go
opts := []ddtrace.FinishOption{tracer.WithError(err)}
if tq.params.config.noDebugStack {
    opts = append(opts, tracer.NoDebugStack())
}
span.Finish(opts...)
```

Reply:
I didn't go with this design because it'll cause an additional allocation, which adds a bit more cpu time & usage, if we were to add more or all finish options later I would agree it'll look better.
```go
func f(...int) {}

func callWithOpts1() {
	opts := []int{}
	opts = append(opts, 1)
	f(opts...)
}

func callWithOpts2() {
	f(1)
}

func BenchmarkOpts(b *testing.B) {
	b.Run("1", func(b *testing.B) {
		for i := 0; i < b.N; i++ {
			callWithOpts1()
		}
	})
	b.Run("2", func(b *testing.B) {
		for i := 0; i < b.N; i++ {
			callWithOpts2()
		}
	})
}
```

```
BenchmarkOpts/1-4 	100000000	         19.8 ns/op	       8 B/op	       1 allocs/op
BenchmarkOpts/2-4 	2000000000	         0.52 ns/op	       0 B/op	       0 allocs/op
```

Reply:
Oh, wow! Indeed! That's a great point! Let's keep it. 👏